### DISCUSSION TITLE
love freeplane! how to customize icon keyboard shortcuts?

### DISCUSSION THREAD

**Discussion by brendanp** [freeplane/freeplane#531] [Help] [answered]
Please help! I'll offer a bounty of $20 via Venmo or paypal or whatever.

Hi! I've been an avid Freeplane user for over a decade (whenever it forked from FreeMind).

I love the "assign hot key function".

However there no longer seems to be a way to **_add icon keyboard shortcuts_** through the UI?

For instance to add a 'star' icon to a node on mac I have:  cmd-I (brings up icon window) +  cmd-A

This used to be possible to set in the preferences window but no longer appears?

[I made a video! ](https://drive.google.com/file/d/1UQMAx9BqwsKnNq0zjAG7tYoYO3vhLW2x/view?usp=drivesdk
![2022-08-03_11-59-08](https://user-images.githubusercontent.com/179787/182677285-145978a6-0ec0-482b-8206-e8b540cca207.png)
)

**Comment by brendanp** [comment]
_this is a comment not an answer_
Sorry for the video coffee shop background noise! Will rerecord that :)

Use case: Using Freeplane for GTD (getting things done) is amazing especially being able to add icons to node through the keyboard.

There no longer seems to be a way to customize keyboard shortcuts for adding icons from the icon table? This used to be available via Preferences.

**Comment by macmarrum** [comment, marked as answer]
I'm not sure how it works on MacOS. On Linux and Window, after I open the Icons Window, I can `Ctrl+click` an icons to assign a hot key. Hot keys for icons will force `Ctrl`, i.e. if I choose `w` as my hot key for the "MS Word" icon, the hot key for it will be `Ctrl+w`.

BTW, `Ctrl+click` works also for menu items, i.e. in additon to **Assign hot key**

In addition to hot keys that work within the Icons Window, Freeplane allows to assign global hot keys to insert an icon (without opening the Icons Window). If you have `Preferences…->Appearance->Icons->Add emojis to menu` set to `on`, `Insert->Icons->Icon by category...` lists the icons. A hot key can bew set by `Ctrl+click` on one of them.

**Reply by brendanp** [reply]
Thank you @macmarrum  ! That indeed works. 

Is that documented somewhere? I did search quite a bit!

Because I forgot GitHub removed private messaging I had to figure out how to send you a private message. I created a github repo and invited you to it where you'll find a pointer to the bounty!  ¯\_(ツ)_/¯

**Reply by macmarrum** [reply]
> Is that documented somewhere? I did search quite a bit!

**Assign hot key** is documented. `Ctrl+click` is missing. Maybe you could join our documentation initiative at https://github.com/freeplane/docs and help to improve the docs?
